### PR TITLE
fix: #56

### PR DESCRIPTION
**Pull request by zhangHongEn**
fix: #56  
fix: #41

### CODE REVIEW

File: examples/vite-vite/vite-host/package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "@swc/core": "~1.7.10",
     "@vitejs/plugin-react": "^4.3.1",
     "vite": "^5.4.0",
-    "vite-plugin-top-level-await": "^1.4.4",
-    "http-server": "^14.1.1"
+    "vite-plugin-top-level-await": "^1.4.4"
```
Comment (husayt) [inline examples/vite-vite/vite-host/package.json:21]: @zhangHongEn do we still need vite-plugin-top-level-await? Vite allows async config like this:

```
export default defineConfig(async ({ command, mode }) => {
// do await here
return { }
}
```
can we not use that?

Comment (zhangHongEn) [reply examples/vite-vite/vite-host/package.json:21]: This is browser compatibility
https://v8.dev/features/top-level-await

Comment (husayt) [reply examples/vite-vite/vite-host/package.json:21]: I see.

**Review by husayt — APPROVED** [review]

**Review by gioboa — APPROVED** [review]
:rocket: @zhangHongEn

**Pull request merged**